README.md

```diff
@@ -118,7 +141,12 @@ entities (no need to draw an ERD)...
 Your answer...
 
 ```
-Replace this with your answer
+The ERD is a graphical representation of an information system that shows the relationship between people, objects, places, concepts or events within that system.
+
+Genie - person, lives in a bottle, grants wishes
+Lamp - may house a genie. My provide light and/or decoration
+Person - name, age, gender
+pet - animal, dog, cat, snake
 ```
```

Comment:
To further your answer: 
Sample relationships...
- A one-to-one relationship between Genie and Lamp
- A one-to-many relationship between Person and Pet